### ISSUE TITLE
Website Carousel Text A/B Test

### ISSUE THREAD

**Issue by majkush** [hackforla/access-the-data#227]
### Overview
The carousels for the website prototype need to be updated due to the transition away from data-literacy modules that is taking place in the project. An A/B test of the carousels will be performed to optimize their usability. 

### Action Items
- [x] Gather the overarching goal for the carousels and the total number of carousels needed
- [x] Define a goal for each carousel
- [x] Craft a research plan for the A/B test with a UX Researcher
- [x] Create a Google Doc to house copy
- [x] Draft version 1 copy for each carousel
- [x] Draft version 2 copy for each carousel
- [x] Get feedback from team, stakeholders, and the organization
- [x] Give to designer to add copy to Figma for web and mobile prototypes
- [x] Get approval for research plan
- [x] Set up the study
- [x] Recruit participants
- [x] Execute study
- [x] Evaluate results
- [x] Report results and suggestions for which carousels were preferred
- [x] Work with a designer to have the simple carousels updated into the latest version of the Figma.

### Resources/Instructions
- [UX Writing Resources](https://github.com/hackforla/access-the-data/wiki/ATD-Website-Overview#ux-writing-resources)
- [Current website prototype for desktop](https://www.figma.com/file/nS5IqnutUT2FuPwwJObHaY/Access-the-Data-Main-Figma?node-id=2%3A16)
### Completed Work
- [website carousel text draft](https://docs.google.com/document/d/17_ZXQ-Q4ibY2LI665snxEVDDZrYwgeSeBNGDNBnTbj8/edit?usp=sharing)
- [Carousel A/B Test Research Background and RQs](https://docs.google.com/document/d/1-lokQLnhewLF0vaLCyBKkix9u_5EsqglgM_RkMQFQaw/edit?usp=sharing)
- [Presentation slides on findings](https://docs.google.com/presentation/d/1WYAwBV5zjsDQTqKuZGvbjWnGUotDbxlpAXVNynBsCkE/edit?usp=sharing)
- [Open-ended responses about how we can improve accessibility to open-access data](https://docs.google.com/spreadsheets/d/1y4WcgZXf6KlZ6fyp8Px8yQrz1PC7w7idgT3ElvLEUfg/edit?usp=sharing)

### Key Takeaways
- Through this research and a team conversation of results at the 11/30/2022 Demo Day ATD meeting, we determined that the Simple carousels will be used in the product. They will be updated into the hi-fi wireframes.

**Comment by majkush** [comment]
Note: The action items are being developed. What is on the issue at the moment is a draft. I will meet with Lucy to discuss further in the next few days.

**Comment by majkush** [comment]
@steffiekim and I are making progress. We met and discussed the first draft of the text. Steffie suggested adjusting the text so that it would work better with the A/B test design method. As a result, I created two sets of text, one that reflects the Empowerment brand principle and one that reflects the Simple brand principle. Text was then put into drafts on Figma to create visuals. Steffie is working on setting up the research study.

**Comment by steffiekim** [comment]
Drafted a survey experiment on Qualtrics and shared it with Matt. (preview [here](https://usc.yul1.qualtrics.com/jfe/preview/previewId/0488eac2-ea4f-4004-a54a-d15892e47241/SV_e8TlRFT16zTVizI?Q_CHL=preview&Q_SurveyVersionID=current))
If resources allow, additional questions from a [previous survey](https://docs.google.com/document/d/1c90PcsDjI3UCYdaGh-aHZQ6KjVrDvy1rGxkcXSMAGLE/edit) that wasn't launched can be added.

**Comment by mxajPrice** [comment]
A/B test comments from PMs:

- Would be useful to define "open-access data" at the beginning of the survey
- Can a "back" button be added?
- Where it asks "I am willing to click on "Learn more"", switch phrase to "Learn more button" because in the screenshot it is a little hard to read "Learn more."


Question: How many participants are you looking to fill this out?

**Comment by steffiekim** [comment]
> A/B test comments from PMs (@lrchang2 , @bkong , @mxajPrice):
>
> * Would be useful to define "open-access data" at the beginning of the survey
> * Can a "back" button be added?
> * Where it asks "I am willing to click on "Learn more"", switch phrase to "Learn more button" because in the screenshot it is a little hard to read "Learn more."
>
> Question: How many participants are you looking to fill this out?

Hi aj, here are the answers:
1. I added the definition at the beginning of additional questions after the experiment. Here's @majkush's definition: Open-access data refers to data that is freely available for anyone to access, use, download, analyze, copy, or redistribute.
2. Yes. The back button is added now.
3. Relevant items are modified.
4. We're aiming for at least 40 participants. But as always, the more, the better.

**Comment by lrchang2** [comment]
Potential Recruitment Strategies:

**Comment by steffiekim** [comment]
Finalized recruitment language and confirmed with @lrchang2:
:wave: Hi! We are UX researchers from Hack for LA working on developing LA City’s website for open-access data.
:globe_with_meridians: Please help us make open-access data easily accessible to all by taking a short 3-minute survey!
:bangbang: Here is the survey [insert link]. Thanks!
P.S. If you know someone else who may be interested, please share this survey with them. :slightly_smiling_face:

**Comment by steffiekim** [comment]
Posted recruitment messages to 3 subreddits and 4 UX communities on Slack.

**Comment by steffiekim** [comment]
Reposted recruitment messages on 2 subreddits (r/SampleSize, r/SurveyExchange) and 3 Slack servers (Hexagon UX, mixedmethods, ResearchOps) since they allow reposts.

**Comment by steffiekim** [comment]
Finished data analysis. Currently making slides for presentation.

**Comment by bkong** [comment]
@steffiekim @majkush Can you update this issue with links to the A/B test results and the presentation you gave on demo day? Also, please outline next steps or blockers you're waiting to resolve here.

**Comment by majkush** [comment]
@bkong Absolutely. 

@steffiekim and I added a section titled 'completed work' in the issue where we listed our completed work, including the presentations and the Excel spreadsheet of qual data. I will reach out to @deepamohan22 early next week to seek her assistance in getting the approved carousels imported into the latest Figma prototype files. 

After that is complete, the issue will be complete and I will close the issue.
